src/lib/GRPCClient.ts

```diff
@@ -0,0 +1,68 @@
+import { loadSync, PackageDefinition } from '@grpc/proto-loader'
+import { Client, credentials, loadPackageDefinition } from 'grpc'
+
+export class GRPCClient {
+	private packageDefinition: PackageDefinition
+	private client: Client
+	private listNameMethods: any[]
+
+	constructor(
+		protoPath: string,
+		packageName: string,
+		service: string,
+		host: string,
+		options = {} as any,
+		tls: boolean = false
+	) {
+		this.packageDefinition = loadSync(protoPath, {
+			defaults: options.default === undefined ? true : options.default,
+			enums: options.enums === undefined ? String : options.enums,
+			keepCase: options.keepCase === undefined ? true : options.keepCase,
+			longs: options.longs === undefined ? String : options.longs,
```

Comment:
This is good but since we can get `string` or `number`, interfaces should be adapted (with generic or by using conditional typing) if we want to add this new feature.

Reply:
@jwulf What do you think ?